### ISSUE TITLE
Provide a download button for the Adoptium Temurin JDK/JRE builds for promotional purposes

### ISSUE THREAD

**Issue by jonelo** [adoptium/adoptium.net#294]
This is a request for the marketing department probably.

Oracle allows users to promote Oracle JDK builds with an official download button logo. See also
https://www.oracle.com/java/java-affinity/download-buttons.html

I think it would be great if Adoptium could provide something similar for their builds.  I would love to promote https://adoptium.net/ if I could use an official download button.  However, I have not found an official download button on the adoptium.net website that I could use for this purpose.

**Comment by karianna** [comment]
I'm not sure this will work per see. Adoptium.net will be transitioning to a marketplace (where you can download AQAVit and TCK compliant OpenJDK binaries from multiple vendors.

We could maybe add a button purely for the Eclipse Temurin download but then which version/OS would it pick?

**Comment by jonelo** [comment]
The marketplace sounds like a great idea to me actually.  Well, in that case you could provide not just one button, but multiple different buttons that promote particular OpenJDK offerings on Adoptium if the marketplace is established.

For now I would be happy to use a download button on my websites that promotes the Eclipse Temurin onestop at adoptium.net.

**Comment by sxa** [comment]
Hmmm I quite like the idea of a Temurin button that would go to either https://adoptium.net/temurin/ or the full page of options at https://adoptium.net/temurin/releases.

**Comment by tellison** [comment]
> Hmmm I quite like the idea of a Temurin button that would go to either https://adoptium.net/temurin/ or the full page of options at https://adoptium.net/temurin/releases.

This is an ideal task for EF Marketing, let's push it to them

**Comment by shanda-eclipse** [comment]
@meKokabi can we prepare some buttons like https://www.oracle.com/java/java-affinity/download-buttons.html for Eclipse Temurin download? See https://github.com/adoptium/adoptium/wiki/Adoptium-Project-Logos for Temurin logo

Timeline: first drafts can be expected next week.

**Comment by meKokabi** [comment]
> 

👍

**Comment by jonelo** [comment]
Good job!  Thank you for the artwork!  I like the transparancy which helps to increase acceptance on various websites.  Is it possible to get different sizes for the .png files?  Are you planning also .svg?  I can hardly wait, but I also wonder what the conditions are for using the buttons. :-)

**Comment by meKokabi** [comment]
Absolutely! Once you choose your desired one, I will make different sizes and formats including svg.

**Comment by sxa** [comment]
I'll circulate these amongst the PMC to make them aware. They look good to me although my only thought is whether it's worth having one that's a bit wider than B (Maybe with the top and bottom cropped down a bit)

**Comment by tellison** [comment]
A few comments:
 - They don't shrink down particularly well as there is some small writing on them.
 - Without a border they don't much look like buttons, maybe get creative and consider a 3D effect for one design?
 - It would also be good to have some options that are different shapes (e.g. longer and shorter), and one or two that work for light/dark/transparent backgrounds.

Just rendering them here for easier viewing...

**Button A**: 
[<img src="https://github.com/tellison/test/raw/main/buttonA.png" width="100"/>](https://adoptium.net/temurin/releases/) 


**Button B**: 
[<img src="https://github.com/tellison/test/raw/main/buttonB.png" width="120"/>](https://adoptium.net/temurin/releases/)


**Button D**: 
[<img src="https://github.com/tellison/test/raw/main/buttonD.png" width="100"/>](https://adoptium.net/temurin/releases/)

**Comment by smlambert** [comment]
I would like to see an option where the rounded edges of the button matches the rounded edges of the logo.  A has none (which I find jarring), and D seems like it has a bit too much rounding (but that may be an illusion because of the mono-colour and no border or shading).  

D is my preferred choice, but I wonder about taking the top diagonal of the Temurin logo and laying it down horizontally as the button.  Would be nice to see if it could match the direction of colour gradient of that logo, rather than no gradient in D or what A and B do where gradient is top to bottom.

**Comment by jerboaa** [comment]
I like 1 and 5. 5 could probably work better without the drop shadow. My $0.02.
